src/ontology/mondo-edit.obo

```diff
@@ -149883,8 +149898,6 @@ name: Lown-Ganong-Levine syndrome
 def: "Lown-Ganong-Levine syndrome is an extremely rare conduction disorder characterized by a short PR interval (less than or equal to 120 ms) with normal QRS complex on electrocardiogram associated with the occurrence of episodes of atrial tachyarrythmias (e.g. atrial fibrillation, atrial tachycardia)." [Orphanet:844]
 subset: gard_rare {source="GARD:16550", source="MONDO:GARD"}
 subset: nord_rare {source="MONDO:NORD"}
-subset: ordo_disorder {source="Orphanet:844"}
```

Comment:
These are the removals we are looking for based on the Mondo QC errors from the previous pipeline run (https://github.com/monarch-initiative/mondo/actions/runs/16376653241/job/46278152823#step:4:39) that was based on only a mini build of data from Mondo Ingest that included updated EMC content only,